netbeans.apache.org/src/content/tutorials/nbm-filetemplates.adoc

```diff
@@ -225,7 +225,7 @@ Once you have defined the file template, the description file, and the icon, you
 
 
 [start=1]
-1. Right-click the module in the Projects window, choose Properties, and use the Libraries tab to add dependencies on Datasystems API and Utilities API.
+1. Right-click the module in the Projects window, choose Properties, and use the Libraries tab to add dependencies on File Templates and Base Utilities API.
 
 [start=2]
 1. 
```

Comment:
@ebarboni I am wondering why the numbered list isn't working (https://netbeans.apache.org/tutorials/nbm-filetemplates.html)

Does the text have to be on the same line as the `1.` ? The 1 appears twice right now, then it skips to 3.

Reply:
looking through the file seems to confirm this. The numbered list is broken everywhere a newline after the `1.` was added.

@SeanCarrick do you want to fix this too? :)

Reply:
I'll get on it today…

Reply:
@mbien , 

Check PR #645 for the numbering fixes.

-SC

Reply:
@SeanCarrick awesome thanks. merged this PR here.